packages/blockchain-wallet-v4/src/types/KVStoreEntry.js

```diff
@@ -163,36 +161,41 @@ export const verifyResponse = curry((address, network, res) => {
   return Either.of(assoc('compute_new_magic_hash', magic(pB, mB, network), res))
 })
 
-export const extractResponse = curry((encKey, encKeyPadded, res) => {
+export const extractResponse = curry((encKey, encKeyUnpadded, res) => {
   if (res === null) {
     return res
   } else {
     // TODO: remove when redux/core is moved to frontend
     if (window.logLevel === 'verbose' && res.type_id !== -1) {
       // eslint-disable-next-line no-console
-      console.info(
-        'LOG: ',
-        encKey
-          ? compose(decrypt(encKey), prop('payload'))(res)
-          : compose(BufferToString, B64ToBuffer, prop('payload'))(res)
-      )
+      try {
```

Comment:
Is this just for debugging or will it be part of the final PR

Reply:
It logs when users login with that special log-level

Reply:
I could probably remove it now since the reason we put it there was to debug what was probably this issue.

Reply:
Since we're failing hard again on failure and this should fix the issue I'd say we can kill it again. But I'm fine either way.